templates/openstack/architecture.html

```diff
@@ -0,0 +1,265 @@
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% block content %}
+  <section class="p-section--hero">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>OpenStack architecture</h1>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>
+            The best recipe for a successful cloud deployment is to follow a proven design guide from an experienced and trusted advisor. Canonical's reference architecture for OpenStack has been successfully deployed across hundreds of customers and thousands of sites.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a class="p-button--positive"
+           href="https://ubuntu.com/engage/dell-openstack-reference-architecture">Check the reference architecture</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Is OpenStack complex?</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <p>
+            Everyone who has any experience with OpenStack knows this feeling very well. You want to try it. You're committed to learning it. You even start exploring its documentation. But then, almost immediately, <a href="https://docs.openstack.org/install-guide/_images/openstack-arch-kilo-logical-v1.png">you are exposed to its complex architecture</a>.
+          </p>
+          <p>
+            Tens of services, hundreds of processes, thousands of different configuration options: all of that to meet the needs of the most demanding industries, end users and their use cases. But do you really need all this overhead? What if all you need is your own general-purpose cloud?
+          </p>
+          <p>Keep reading to learn how Canonical OpenStack helps you bypass all this complexity.</p>
+        </div>
+      </div>
+      <div class="p-image-container u-hide--small u-hide--medium">
+        {{ image(url="https://assets.ubuntu.com/v1/943a8b94-is-openstack-complex-img.png",
+                alt="",
+                width="1280",
+                height="534",
+                hi_def=True,
+                attrs={"class": "p-image-container__image"},
+                loading="lazy") | safe
+        }}
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>Why Canonical OpenStack?</h2>
+      </div>
+      <div class="col p-section">
+        <p>
+          If you want to use OpenStack, but don't necessarily want to deal with its internals, you're in the right place. Canonical effectively tames the complexity of the upstream OpenStack project and delivers its distilled excellence in the form of a human-friendly product &mdash; Canonical OpenStack.
+        </p>
+        <hr class="p-rule--muted" />
+        <a href="/openstack">Explore Canonical OpenStack&nbsp;&rsaquo;</a>
```

Comment:
Design wants us to wrap these in p elements always so that they have the right spacing. This should also match the figma more closely 
```suggestion
        <p><a href="/openstack">Explore Canonical OpenStack&nbsp;&rsaquo;</a></p>
```